definitions/variable/technology/electricity-grid.yaml

```diff
@@ -28,13 +28,27 @@
 - Network|Electricity|Active Power Flow:
     description: Active power flow through a line
     unit: MW
+- Network|Electricity|Active Power Flow Generation Contribution:
```

Comment:
Is it really possible to identify a specific generator is contributing? I understand from your explanation that this is the share of power flow in the interconnection which is coming from a specific region (maybe produced in yet another one, bought and then sold through this interconnector?) I would remove the word Generation and keep only Active Power flox contribution.  (if I have understood well your explanation), and write in the description sth like 'share of the flow in the interconnection region1>region2 which is sent from region3